Sources/ShopifyCheckoutSheetKit/CheckoutDelegate.swift

```diff
@@ -39,6 +39,9 @@ public protocol CheckoutDelegate: AnyObject {
 	/// This includes email address or telephone number via `mailto:` or `tel:` or `http` links directed outside the application.
 	func checkoutDidClickLink(url: URL)
 
+	/// Tells the delegate that the checkout pay button was touched
+	func checkoutDidTouchPayButton()
```

Comment:
What's the reasoning for exposing this lifecycle method if it doesn't control the haptic feedback?

I would expect this method to fire the haptic feedback, but allow consumers to override it by implementing the method